README.md

```diff
@@ -1,8 +1,10 @@
 # sounDMR <img src="man/figures/logo.png" align="right" height="120" alt="" />
-For more information on the package, as well as examples of the types of downstream analyses that can be run using these results see Colicchio et al., 2023, Scientific Reports : https://www.nature.com/articles/s41598-023-38356-7
+For more information on the package, as well as examples of the types of downstream analyses that can be run using these results see Colicchio et al., 2023, [Scientific Reports](https://www.nature.com/articles/s41598-023-38356-7)
 
 The following document contains a walkthrough of how to get ONT data into the appropriate format to run through sounDMR and a simple tutorial of the functions used to run a differential methylation analysis using sounDMR.
 
+P.S : The whole genome bedmethyl analysis is still in its beta stage. We are currently building a robust code to handle large data and will be releasing the full version soon.  
```

Comment:
I think instead of writing "P.S." I would put something like "Note:". Sounds a little more professional. The rest of it sounds good though!